static/allinone.css

```diff
@@ -64,10 +64,12 @@ h4 {
 
 .navul {
     list-style: none;
-    margin: 10px;
+    /* margin: 10px; */
+    /* margin-bottom: 30px; */
```

Comment:
If you delete this margins it corrupt the navigation bar 
Your Change:
![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/52cb30f2-109b-4355-bd71-61139bf2ff65)
Stable:
![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/8c067ca9-8dcc-4bdb-89ac-0abfdda1dbac)